main.tf

```diff
@@ -17,126 +21,194 @@ resource "aws_s3_bucket" "default" {
   #bridgecrew:skip=BC_AWS_S3_16:Skipping `Ensure S3 bucket versioning is enabled` because dynamic blocks are not supported by checkov
   #bridgecrew:skip=BC_AWS_S3_14:Skipping `Ensure all data stored in the S3 bucket is securely encrypted at rest` because variables are not understood
   #bridgecrew:skip=BC_AWS_GENERAL_56:Skipping `Ensure that S3 buckets are encrypted with KMS by default` because we do not have good defaults
-  count               = local.enabled ? 1 : 0
-  bucket              = local.bucket_name
-  acl                 = try(length(var.grants), 0) == 0 ? var.acl : null
-  force_destroy       = var.force_destroy
-  tags                = module.this.tags
-  acceleration_status = var.transfer_acceleration_enabled ? "Enabled" : null
-
-  dynamic "versioning" {
-    for_each = var.versioning_enabled ? [true] : []
+  count         = local.enabled ? 1 : 0
```

Comment:
```suggestion
  #bridgecrew:skip=BC_AWS_GENERAL_72:Skipping `Ensure S3 bucket has cross-region replication enabled` because this is configurable via var.s3_replication_rules.
  count         = local.enabled ? 1 : 0
```

Reply:
@Nuru  what do you think about this suggestion?